pkg/controllers/kube-apiserver.go

```diff
@@ -165,7 +165,7 @@ func (s *KubeAPIServer) configure(cfg *config.MicroshiftConfig) error {
 									APIVersion: "route.openshift.io/v1",
 									Kind:       "HostAssignmentAdmissionConfig",
 								},
-								Domain: cfg.Cluster.Domain,
+								Domain: cfg.BaseDomain, // @benluddy, would this be the right thing?
```

Comment:
@benluddy ^ would this be the right thing ?

Reply:
Missing `cfg.ClusterName`